packages/cli/src/networks/index.ts

```diff
@@ -77,6 +78,63 @@ export async function fetchBootnodes(network: NetworkName): Promise<string[]> {
   return enrs;
 }
 
+/**
+ * Reads and parses a list of bootnodes for a network from a file.
+ */
+export async function readBootnodes(bootnodesFilePath: string): Promise<string[]> {
+  if (!fs.existsSync(bootnodesFilePath)) {
```

Comment:
since this shouldn't be default, may be this should throw error (in that case readFileSync should directly throw error)

Reply:
Ah yeah, I was wondering about this and if we should just bail out instead of just telling the user "sorry, couldn't do anything". Sure then, we can just let `readFileSync` throw!

Reply:
>Should we also throw if no bootnodes are found in a readable file?


yes i think that would be a more predictable behavior

Reply:
Done in https://github.com/ChainSafe/lodestar/pull/3562/commits/71c3af90156f9402ad466aba6ff4693dd72aac99  @g11tech. Happy to iron out this error behavior further so we're 100% happy with it!